src/Sentry/Internal/BatchProcessorTimer.cs

```diff
@@ -0,0 +1,61 @@
+using System.Timers;
+
+namespace Sentry.Internal;
+
+internal abstract class BatchProcessorTimer : IDisposable
```

Comment:
This whole file for me seems like boilerplate.

It's just wrapping the .NET Timer while giving the same API?
Either make it better, (methods when side effect happens, skip the events, get Sentry diag logging, etc) or delete this altogether